modules/internal/ChapelRange.chpl

```diff
@@ -700,6 +700,9 @@ module ChapelRange {
 
   pragma "no doc"
   inline proc range.alignedLowAsInt {
+    if isAmbiguous() {
```

Comment:
Close.  It's true if the range is strided and the stride starts from a bound that isn't well defined.  So `..n` and `1..` and `..` are unambiguous because there's no stride, so the alignment is trivial (all indices are defined by the range).  And `1.. by 2` and `..10 by -2` are unambiguous because they start from a well defined bound (so the first defines odd integers 1 and higher, and the second defines even integers 10 and lower).  But `..10 by 2` and `1.. by -2` and `.. by 2` are ambiguous because the stride indicates counting from a bound that doesn't exist, so it's not clear whether the alignment is odd or even.